### PR TITLE
Unit tests coverage

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala [hunk 1]
```diff
@@ -161,7 +177,8 @@ class CheckAllAnswersAndSubmitControllerSpec
 
         "display the page" when {
 
-          def test(sessionData: SessionData, expectedTitleKey: String, userType: Option[UserType]): Unit = {
+          def test(sessionData: SessionData, expectedTitleKey: String, userType: Option[UserType], isATrust: Boolean)
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala:180]: do you need the `isATrust` flag here - shouldn't what we had before work with `ompleteFillingOutReturn.subscribedDetails.isATrust`? That is essentially what you're passing in anyway

Comment (rambabu-posa) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala:180]: It works for first test as it's just using existing `completeFillingOutReturn`. But does not for next test. as we are modifying `completeFillingOutReturn` with  `userType'

Comment (rambabu-posa) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala:180]: it took me time to figure out this issue

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/exemptionandlosses/ExemptionAndLossesControllerSpec.scala [hunk 2]
```diff
@@ -1809,43 +1809,4 @@ object ExemptionAndLossesControllerSpec extends Matchers {
     )
   }
 
-  def validateExemptionAndLossesCheckYourAnswersPage(
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/exemptionandlosses/ExemptionAndLossesControllerSpec.scala:1812]: wasn't this being called in `ExemptionAndLossesControllerSpec` and the spec for the CheckAllAnswersController as well? If they weren't they probably should have been to check that the correct content is being displayed

Comment (rambabu-posa) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/exemptionandlosses/ExemptionAndLossesControllerSpec.scala:1812]: Nope, just copied same function and added one extra param "isAnAgent: Boolean".
You can see that new function just above.
All are using this new function.

**Pull request merged**